Wikibase/Tests/Commenting/RedundantVarName.php.expected

```diff
@@ -1,3 +1,3 @@
- 31 | ERROR | [x] Found redundant variable name in @var
- 36 | ERROR | [x] Found redundant variable name in @var
- 41 | ERROR | [x] Found redundant variable name in @var
+ 31 | ERROR | [x] Found redundant variable name "$redundantVariableName" in @var
```

Comment:
So much more readable! 👏

(That’s an “applause” :clap: and not a Twitter-style “make :clap: this :clap: better” :clap: 😜  )